### ISSUE TITLE
ImportError: libsvml.so: cannot open shared object file: No such file or directory

### ISSUE THREAD

**Issue by vlad-nazarov** [intel/scikit-learn-intelex#1098]
**Describe the bug**
Importing sklearnex 2023.0 version fails with ImportError.  There is an additional dependency on Intel Compiler. Sourcing BaseToolkit solves the problem.

**To Reproduce**
1. Download from pip latest skleanex package
2. python -c "import sklearnex"

Found the same problem in [your pipelines](https://dev.azure.com/daal/daal4py/_build/results?buildId=21250&view=logs&j=82fe0a7a-6695-5b19-cfd7-83d8770eb236&t=18ea527f-bd9f-546f-19c1-76982f4765de&l=20)

**Expected behavior**
Sklearnex imported without ImportError

**Output/Screenshots**
Traceback (most recent call last):
  File "<string>", line 1, in <module>
  File "/home/user/miniconda3/envs/scklearnex_test/lib/python3.9/site-packages/sklearnex/__init__.py", line 18, in <module>
    from .dispatcher import patch_sklearn
  File "/home/user/miniconda3/envs/scklearnex_test/lib/python3.9/site-packages/sklearnex/dispatcher.py", line 22, in <module>
    from daal4py.sklearn._utils import daal_check_version, sklearn_check_version
  File "/home/user/miniconda3/envs/scklearnex_test/lib/python3.9/site-packages/daal4py/__init__.py", line 38, in <module>
    from daal4py._daal4py import *
ImportError: libsvml.so: cannot open shared object file: No such file or directory

**Environment:**
- WSL 2 with Ubuntu 20.04
- Python 3.9 from conda
- scikit-learn-intelex  2023.0.0 pypi_0

**Comment by napetrov** [comment]
Fixed already, and would be hopefully fixed on pypi soon enough. 

Another workaround would be to install Compiler runtime
pip install dpcpp-cpp-rt

**Comment by vlad-nazarov** [comment]
Thanks, this workaround works! Issue can be closed

**Comment by napetrov** [comment]
Will keep it here open until release deployment. Might help other people

**Comment by ystoll** [comment]
Yes, I can confirm that @napetrov answer is useful.

**Comment by ystoll** [comment]
Hi again, I need to install scikit-learn-intelex as a dependency for a project.
Until then, I added dpcpp-cpp-rt as well as mentioned above to avoid import error. 
The problem is that one of my collaborator, working on a mac, has got some trouble with this package, hence I would like to install the package directly from github until such time where the patched version will be available on pip.

When trying to do that, I have an issue from pip:

```console
(.venv310) (py310) user@host:sv_regression$ pip install git+https://github.com/intel/scikit-learn-intelex.git@master#egg=scikit-learn-intelex
Collecting scikit-learn-intelex
  Cloning https://github.com/intel/scikit-learn-intelex.git (to revision master) to /tmp/pip-install-7nir3kc4/scikit-learn-intelex_f45b5f5ff08e475db564879eb34a7068
  Running command git clone --filter=blob:none --quiet https://github.com/intel/scikit-learn-intelex.git /tmp/pip-install-7nir3kc4/scikit-learn-intelex_f45b5f5ff08e475db564879eb34a7068
  Resolved https://github.com/intel/scikit-learn-intelex.git to commit ce952e53e355715e04a09e39c9959fa6c7bcbb15

  Preparing metadata (setup.py) ... error
  error: subprocess-exited-with-error
  
  × python setup.py egg_info did not run successfully.
  │ exit code: 1
  ╰─> [6 lines of output]
      Traceback (most recent call last):
        File "<string>", line 2, in <module>
        File "<pip-setuptools-caller>", line 34, in <module>
        File "/tmp/pip-install-7nir3kc4/scikit-learn-intelex_f45b5f5ff08e475db564879eb34a7068/setup.py", line 46, in <module>
          raise RuntimeError("Not set DALROOT variable")
      RuntimeError: Not set DALROOT variable
      [end of output]
  
  note: This error originates from a subprocess, and is likely not a problem with pip.
error: metadata-generation-failed

× Encountered error while generating package metadata.
╰─> See above for output.

note: This is an issue with the package mentioned above, not pip.
hint: See above for details.

```


Does anyone have a workaround ? Thanks.

**Comment by Alexsandruss** [comment]
It's expected because repository doesn't fully support this way of installation now.
At least, you need to set all dependencies from [daal4py/INSTALL.md](https://github.com/intel/scikit-learn-intelex/blob/master/daal4py/INSTALL.md). They are listed on requirements-dev.txt except C++ compiler, clang-format, MPI, TBB and oneDAL.
Last three can be installed with `pip install dal-devel impi-devel tbb-devel` and set with `export {DAL/TBB/MPI}ROOT={your python env root}`.
*Theoretically*, you can use `pip install git+...` with set dependencies to install daal4py only. setup_sklearnex.py which installs sklearnex will not be called in that case.

If dependencies are set, it's simpler to clone repo and run following daal4py+sklearnex installation commands:
```bash
python setup.py install --single-version-externally-managed --record=record1.txt
python setup_sklearnex.py install --single-version-externally-managed --record=record2.txt
```
or
```bash
python setup.py develop --no-deps
python setup_sklearnex.py install develop --no-deps
```

**Comment by napetrov** [comment]
Fixed version have been deployed to PyPi - 2023.0.1
